### PR TITLE
feat(Twitter):#21 Integrated Twitter.

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
1. Used `angular/fire` modules to integrate twitter.
2. Fixed the lint errors.
Screen Shot 2019-04-11 at 1 55 33 PM

### CODE REVIEW

File: web/src/app/services/auth.service.ts [hunk 1]
```diff
@@ -12,4 +12,10 @@ export class AuthService {
       new firebase.auth.GithubAuthProvider()
     ));
   }
+
+  signInWithTwitter() {
```
Comment (eddiejaoude) [inline web/src/app/services/auth.service.ts]: Missing return type

Comment (eddiejaoude) [reply web/src/app/services/auth.service.ts]: I think it should be `Observable<UserCredential>`

https://firebase.google.com/docs/reference/js/firebase.auth#usercredential

Comment (VenkatVenkatesh) [reply web/src/app/services/auth.service.ts]: Will check it out. :+1:

**Comment by eddiejaoude** [comment]
@VenkatVenkatesh did you mean to close this?

**Comment by VenkatVenkatesh** [comment]
Apologies Eddie, It was a network issue. I unknowingly closed the PR 😄 .

**Comment by eddiejaoude** [comment]
🤓

File: web/src/app/services/auth.service.ts (hunk 1 above)
Comment (VenkatVenkatesh) [reply web/src/app/services/auth.service.ts]: shall I do this change in the current branch which I am working on, as it may break the task flow.

Comment (eddiejaoude) [reply web/src/app/services/auth.service.ts]: Normally not really, as it will go into `develop`, but for now its fine 👍

**Comment by eddiejaoude** [comment]
@VenkatVenkatesh Can you run the lint checker locally please before committing and pushing 🤓

**Review by eddiejaoude — APPROVED** [review]

**Comment by VenkatVenkatesh** [comment]
Sure Eddie, I am already doing it. Will double check it hereafter 👍

**Comment by eddiejaoude** [comment]
Thanks 👍

**Pull request merged**